Ghidra/Features/Base/ghidra_scripts/ExportFunctionInfoScript.java

```diff
@@ -192,10 +175,9 @@ private void exportMetricsJson(File outputFile, AddressSetView scope) throws IOE
 				String entry = f.getEntryPoint().toString();
 				long size = body.getNumAddresses();
 
-				// Instruction count
+				// Instruction count - use for-each style loop
 				int instCount = 0;
-				for (Iterator<Instruction> insIt = listing.getInstructions(body, true); insIt.hasNext();) {
-					insIt.next();
+				for (Instruction inst : listing.getInstructions(body, true)) {
```

Comment:
This introduces a new warning because `inst` is unused.  It was probably better the way it was before.  Even better, you can try using a stream...something like `long instCount = CollectionUtils.asStream(listing.getInstructions(body, true)).count()`.  I haven't tested that though...you'd need to verify.